test/Microsoft.Azure.ServiceBus.UnitTests/Microsoft.Azure.ServiceBus.UnitTests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft.Azure.ServiceBus.UnitTests</AssemblyTitle>
     <TargetFrameworks>netcoreapp1.0;net46</TargetFrameworks>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Hmmm, in general, warnings should be treated as errors and exceptions should be excluded with pragmas :wink: